runtime/libffi/module.xml

```diff
@@ -84,6 +84,9 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 			<makefilestub data="FFI_TARGET_DIR := z">
 				<include-if condition="spec.zos_390.*" />
 			</makefilestub>
+			<makefilestub data="FFI_TARGET_DIR := riscv">
+				<include-if condition="spec.linux_riscv64.*" />
+			</makefilestub>
```

Comment:
Please put these additions in their place so the stubs continue to ordered by spec name.
Likewise with the stubs below and remove trailing whitespace.

Reply:
Fixed as suggested above.